<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_drivers</class>
 <widget class="QWidget" name="edit_drivers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_born">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_vod_ud">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kat_prav">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_data_vidachi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_data_finish">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>520</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_fio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата рождения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Телефон</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер водительского удостоверения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Категория прав</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_data_vidachi">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата выдачи прав</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата окончания прав</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_poisk">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите ФИО, которое хотите исправить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_poisk">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
